app/views/osbbs/show.html.slim

```diff
@@ -11,6 +11,11 @@
       .info_item
         span.nameing = t('.osbb-email')
         span.insert = @osbb.email
+      .info_item
+        span.nameing = t('.osbb-address')
```

Comment:
Problems with translation

Reply:
What a problems?

Reply:
On screenshot Osbb Address but in file locales/en.yaml Address

Reply:
![Screenshot from 2020-10-20 15-30-25](https://user-images.githubusercontent.com/31774981/96586138-3c1ac780-12e9-11eb-909d-8c1d0a6164b9.png)
not exist ukraine translation for osbb-address